language/stdlib/modules/ValidatorConfig.move

```diff
@@ -314,11 +322,18 @@ module ValidatorConfig {
         apply OperatorRemainsSame to * except set_operator, remove_operator;
     }
 
-    /// LIP-6 Property: If address has a ValidatorConfig, it has a validator role.  This invariant is useful
-    /// in LibraSystem so we don't have to check whether every validator address has a validator role.
-    /// <a name="ValidatorConfigImpliesValidatorRole"></a>
     spec module {
-        invariant [global] forall addr1: address where exists_config(addr1):
+        /// LIP-6 Property: If address has a ValidatorConfig, it has a validator role.  This invariant is useful
+        /// in LibraSystem so we don't have to check whether every validator address has a validator role.
+        /// ref: "ValidatorConfigImpliesValidatorRole"
+        invariant [global, isolated] forall addr1: address where exists_config(addr1):
```

Comment:
If this invariant is used to prove other things, maybe we shouldn't use `isolated` here?